src/web3py/contract_tweak.py

```diff
@@ -1,20 +1,30 @@
 import itertools
```

Comment:
Probably it should be ignored because mostly it's just a copypaste from `web3py`

Reply:
Done